packages/rust/rust-src.subpackage.sh

```diff
@@ -0,0 +1,5 @@
+TERMUX_SUBPKG_DESCRIPTION="Rust source code files"
+TERMUX_SUBPKG_PLATFORM_INDEPENDENT=true
+TERMUX_SUBPKG_INCLUDE="
+lib/rustlib/src
```

Comment:
We will need 

```
Replaces: rust (<< 1.74.1-1)
Breaks: rust (<< 1.74.1-1)
```

On all these new subpackages for upgrade to go smoothly.

Reply:
I wonder it this is really necessary since the subpackages will not install as they hard depends the main rust package and specific version

Reply:
Ok I somewhat manage to reproduce the `trying to overwrite ..., which is also in package ...` issue with `dpkg -i` rudimentary install order but not with `apt install` smart resolving order. So leaving it in to be safe.

I will keep in mind on this for future.